pkg/karmadactl/unjoin.go

```diff
@@ -26,7 +26,7 @@ var (
 	unjoinShort   = `Remove the registration of a cluster from control plane`
 	unjoinLong    = `Unjoin removes the registration of a cluster from control plane.`
 	unjoinExample = `
-%s unjoin CLUSTER_NAME --cluster-kubeconfig=<KUBECONFIG>
+%s unjoin member1 --cluster-kubeconfig=$HOME/.kube/members.config
```

Comment:
Does this necessary?

Reply:
I think that Usage has provided a template for command, Example should provide a practical usage.
```
[root@master67 ~]# karmadactl unjoin -h
Unjoin removes the registration of a cluster from control plane.

Usage:
  karmadactl unjoin CLUSTER_NAME --cluster-kubeconfig=<KUBECONFIG> [flags]

Examples:

karmadactl unjoin member1 --cluster-kubeconfig=$HOME/.kube/members.config
```

Reply:
Because of adding --wait Example to `Unjoin`, in order to keep consistent, it is not changed for now.

Reply:
Can we do it in this PR as well?

Reply:
I merged the commit, so there have been --wait Example. Better presentation of Example we could have a more detailed discussion for all sub commands like `karmadactl get` in a new pr.